README.md

```diff
@@ -140,6 +140,22 @@ Run the build and copy the `iframe-resizer` host-side JS into your local `Django
     $ yarn build
     $ yarn copy-iframe-resizer
 
+The front application is tested using [jest](https://jestjs.io/). Every js module has its corresponding spec file containing related tests:
+
+```
+├── VideoPlayer
+│   ├── index.spec.tsx
+│   ├── index.tsx
+│   ├── indexWithDashjs.spec.tsx
+│   └── VideoPlayer.css
+```
+
+To run the test you have to change your directory to `src/frontend` and run `yarn test`
```

Comment:
```suggestion
To run the tests you have to change your directory to `src/frontend` and run `yarn test`
```

Reply:
This sentence is wrong in fact. Previously we already ask the user to mode to `src/frontend` so we only have to show how to run tests. I made the modification if you want to read it.